plotly/version.py

```diff
@@ -1 +1 @@
-__version__ = '2.0.8'
+__version__ = '2.0.9'
```

Comment:
do you want to leave this as 2.0.9?
2.0.8 isn't released yet (was planning on updating on pip when the new plotly.js version pr is merged)